stdlib/2and3/builtins.pyi

```diff
@@ -943,7 +947,10 @@ class list(MutableSequence[_T], Generic[_T]):
     def remove(self, __value: _T) -> None: ...
     def reverse(self) -> None: ...
     if sys.version_info >= (3,):
-        def sort(self, *, key: Optional[Callable[[_T], Any]] = ..., reverse: bool = ...) -> None: ...
+        @overload
+        def sort(self: List[_LT], *, key: None = ..., reverse: bool = ...) -> None: ...
+        @overload
+        def sort(self, *, key: Callable[[_T], _LT] = ..., reverse: bool = ...) -> None: ...
```

Comment:
You should remove the `= ...` from this overload (same with `sorted`).

Reply:
I am sorry, which `= ...` should I remove?

Reply:
Sorry, the one for `key`. In this overload the `key` argument is required.

Reply:
Yes, it took me a few minutes to understand what you meant. Thanks!

I did not touch the Python 2 version of `sort`, but I was surprised to see that neither `cmp` nor `key` are `Optional`... I wonder how that's working now.